uaclient/cli.py

```diff
@@ -140,9 +140,39 @@ def detach_parser(parser):
     return parser
 
 
+def help_parser(parser):
+    """Build or extend an arg parser for help subcommand."""
+    usage = USAGE_TMPL.format(name=NAME, command="help [service_name]")
+    parser.usage = usage
+    parser.prog = "help"
+    parser._positionals.title = "Arguments"
+    parser.add_argument(
+        "service_name",
+        action="store",
+        nargs="?",
+        default="",
+        help="the name of the Ubuntu Advantage service to get help info",
```

Comment:
Just a wording nit:

    a service to view help output for

We should double-check the help output in the status command as well. Over time I believe we'll grow things that are not Ubutun Advantage services and so I think there's value in being more generic in places like this.